mk/spksrc.cross-cc.mk

```diff
@@ -73,7 +73,7 @@ include ../../mk/spksrc.generate-digests.mk
 include ../../mk/spksrc.dependency-tree.mk
 
 .PHONY: all-archs
-all-archs: $(addprefix arch-,$(AVAILABLE_ARCHS))
+all-archs: $(addprefix arch-,$(AVAILABLE_TOOLCHAINS))
```

Comment:
can we use singular such as `AVAILABLE_TOOLCHAIN` ?
and while at it migrate `spksrc/toolchains` to `spksrc/toolchain`

Reply:
No, AVAILABLE_TOOLCHAINS is a list of all available "arch-tc" toolchains.
But good to remember to rename the folder to spksrc/toolchain.